### DISCUSSION TITLE
Play sound for new race-control messages

### DISCUSSION THREAD

**Discussion by FelixSFD** [f1multiviewer/issue-tracker#101] [Feature requests] [answered]
I usually have the RCM window open on my laptop outside of my field of view, as my larger screens are already full with streams. Because of that, I often miss messages like track-limits or "car XX left the track at turn Y and continued".

If would be nice, if we could configure sounds for each type of message to be played. I'm imagining a list of icons (+ maybe a sample message on hover) and a select-field to select one of a few default-sounds. (but just having a checkbox would be fine, too)

**Comment by Psybertr0n** [comment]
It would also be nice to have the latest message flash when it appears in addition to the optional sound chime playing.

**Reply by f1multiviewer** [reply]
That's tracked here: https://github.com/f1multiviewer/issue-tracker/discussions/46

**Comment by f1multiviewer** [comment, marked as answer]
Done!
